spk/python312/crossenv/requirements-duplicity.txt

```diff
@@ -0,0 +1,10 @@
+pip==24.3.1
+setuptools==78.1.0
+wheel==0.45.1
+#
+build:Cython==3.0.11
+build:pip-tools==7.4.1
+#
+cross:build==1.2.2
+cross:cffi==1.17.1
+cross:pipx==1.7.1
```

Comment:
I'm curious about how you derive this file. I don't believe we have this documented fully. If I look at the "_pyproject.toml_" for duplicity, I see the following:

> [build-system]
> requires = [
>     "build>=1.1",
>     "pip>=24.0",
>     "pipx>=1.5",
>     "packaging>=20.0",
>     "setuptools>=78.1.0",
>     "wheel>=0.42.0",
> ]
> build-backend = "setuptools.build_meta"

Would this be used to create the above? If so, how do you determine what should be inserted as-is and what should have a `build:` or a `cross:` prefix?

Reply:
I was thinking of that and i believe @hgy59 also proposed that.  Issue i foud is that would genereate a crossenv for every wheel builds if we were to automate the use of that dependency from the toml file.

And we cannot update a crossenv on demand as some wheels will have conflicting dependencies.